pkgs/servers/mpd/default.nix

```diff
@@ -30,13 +30,17 @@
 
 let
   lib = stdenv.lib;
+  concatAttrVals = nameList: set: lib.concatMap (x: set.${x} or []) nameList;
```

Comment:
If the nativeFeatureDependencies logic will be removed, I would find this wrapper function redundant if used only once :).